anyway/widgets/all_locations_widgets/seriously_injured_killed_in_bicycles_scooter_widget.py

```diff
@@ -47,17 +45,16 @@ def get_seriously_injured_killed_in_bicycles_scooter(
         return res
 
     @staticmethod
-    def create_location_description(location_info: LocationInfo, location_text: str) -> str:
+    def create_location_description(location_info: LocationInfo) -> str:
         return _("in ") + location_info[Constants.YISHUV_NAME] \
             if Constants.YISHUV_NAME in location_info \
-            else location_text
+            else _("In") + " " + location_info["road_segment_name"]
 
     @staticmethod
     def localize_items(request_params: RequestParams, items: Dict) -> Dict:
         subtitle = SeriouslyInjuredKilledInBicyclesScooterWidget.create_location_description(
-            request_params.location_info,
-            request_params.location_text)
-        items["data"]["text"] = {"title": TITLE,
+            request_params.location_info)
+        items["data"]["text"] = {"title": _("Number of severely injured or killed in bike, e-bike, or scooter accidents"),
```

Comment:
Hi,
The (unwritten ) agreement of the `Widget`s is that `items` is being generated in `generate_items()`, and localization is done in `localize_items()`. Future maintainers may not be aware that `create_location_description()` is part of the localization. I think it is best that the logic of creating the description will be performed during the `generate_items()`, and only the localization will be performed during the `localize_items()`.

Reply:
Hi, I moved the code from create_location_description to localize_items

I am not sure if you are saying that we should separate the description logic and the translation further, but if you do I think we should release this widget first and think about it later.

Reply:
Ok, I moved the code back to localize_items